src/data/script.ts

```diff
@@ -233,6 +239,14 @@ export const getActionType = (action: Action): ActionType => {
     return "variables";
   }
   if ("service" in action) {
+    if ("extra" in action) {
+      if (
+        (action as ServiceAction).service === "scene.turn_on" &&
```

Comment:
is this `as` needed since you check `"service" in action` ?

Reply:
Yeah, because we have an unknown action type